Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs

```diff
@@ -373,11 +383,29 @@ private void StartWithChromeV2Debugger(string program, string nodeRuntimeExecuta
             var visualStudioInstallationInstanceID = setupInstance.GetInstanceId();
 
             // The Node2Adapter depends on features only in Node v6+, so the old v5.4 version of node will not suffice for this scenario
-            var pathToNodeExe = Path.Combine(setupInstance.GetInstallationPath(), "\\JavaScript\\Node.JS\\v6.4.0_x86\\Node.exe");
+            var pathToNodeExe = Path.Combine(setupInstance.GetInstallationPath(), "JavaScript\\Node.JS\\v6.4.0_x86\\Node.exe");
+
+            // We check the registry to see if any parameters for the node.exe invocation have been specified (like "--inspect"), and append them if we find them.
```

Comment:
This is the node instance hosting the debugger code.  I'll add a comment that makes that more clear.